client/analytics/settings/config.js

```diff
@@ -24,6 +24,13 @@ const defaultOrderStatuses = [
 	'pending',
 	'on-hold',
 ];
+
+const actionableOrderStatuses = Array.isArray(
+	wcSettings.wcAdminSettings.woocommerce_actionable_order_statuses
```

Comment:
Good catch with this @timmyc! Per the discussion [here](https://github.com/woocommerce/woocommerce-admin/pull/2292#discussion_r287166239), what do you think about cloning the array before the assignment? I guess now that we are checking that it's an array, using the spread operator should be safe:
```suggestion
	[ ...wcSettings.wcAdminSettings.woocommerce_actionable_order_statuses ]
```

Reply:
Cloning seems like a good idea - as for the spread operator, i wasn't sure why we would need it if we verify it is an array.

Reply:
I see that the spread was cloning before 🥇